proto/rill/admin/v1/api.proto

```diff
@@ -733,6 +812,30 @@ message IssueRepresentativeAuthTokenResponse {
   string token = 1;
 }
 
+message RevokeServiceAuthTokenRequest {
+  string token_id = 1;
+}
+
+message RevokeServiceAuthTokenResponse {}
+
+message IssueServiceAuthTokenRequest {
+  string organization_name = 1 [(validate.rules).string.min_len = 1];
+  string service_name = 2;
+}
+
+message IssueServiceAuthTokenResponse {
+  string token = 1;
+}
+
+message ListServiceAuthTokensRequest {
+  string organization_name = 1 [(validate.rules).string.min_len = 1];
+  string service_name = 2;
+}
+
+message ListServiceAuthTokensResponse {
+  repeated string tokens = 1;
+}
```

Comment:
Should return a `ServiceToken` type containing id, created_on, and expires_on. Then we can print these values